src/asmcnc/core_UI/job_go/popups/popup_yetipilot_settings.py

```diff
@@ -416,6 +455,10 @@ def make_option(version_text, version):
         close_button.bind(on_press=unschedule_clocks)
         close_button.bind(on_press=popup.dismiss)
 
+        # if spindle_health_check_button: 
+        spindle_health_check_button.bind(on_press=lambda instance: start_spindle_health_check())
```

Comment:
why lambda instance? 

doesn't matter for now but I think it can be simplified to just on_press=start_spindle_health_check

Reply:
because on_press passes variable instance to the function, so I'm using a lambda function to get rid of it as my start_spindle_health_check function doesn't need the instance